app/code/Magento/Ui/view/base/web/js/form/element/ui-select.js

```diff
@@ -153,8 +153,8 @@ define([
             showPath: true,
             labelsDecoration: false,
             disableLabel: false,
-            filterRateLimit: 500,
-            filterRateLimitMethod: 'notifyAtFixedRate',
+            filterRateLimit: 1000,
+            filterRateLimitMethod: 'notifyWhenChangesStop',
```

Comment:
I don't think these defaults changes are necessary, also this is backward incompatible change